tests/integration/targets/efs/tasks/main.yml

```
@@ -303,6 +303,22 @@
         that:
           - efs_result is not changed
 
+    # ============================================================
+    - name: Update Efs to use elastic throughput_mode
+      community.aws.efs:
+        state: present
+        name: "{{ efs_name }}-test-efs"
```

Looking at the error message from the integration tests, we can't make any more changes to the current volume.  (Notice that the "wait until" date is the day after the test was run):

```
2024-01-11 20:10:09.701316 | controller |     "message": "You have reached the maximum number of throughput mode changes or provisioned throughput value decreases. Wait until 2024-01-12T20:09:02Z UTC and try again.",
2024-01-11 20:10:09.701322 | controller |     "msg": "Unable to update file system.: An error occurred (TooManyRequests) when calling the UpdateFileSystem operation (reached max retries: 4): You have reached the maximum number of throughput mode changes or provisioned throughput value decreases. Wait until 2024-01-12T20:09:02Z UTC and try again.",
```

This test will need to be run using a different EFS volume (I'd suggest creating a fresh volume rather than updating this existing one). 

```suggestion
        name: "{{ efs_name }}-test-efs-2"
```

(please update the always block to also delete  "{{ efs_name }}-test-efs-2")